devito/ir/equations/equation.py

```diff
@@ -220,8 +221,11 @@ def writes(self):
     def xreplace(self, rules):
         return LoweredEq(self.lhs.xreplace(rules), self.rhs.xreplace(rules), **self.state)
 
-    def func(self, *args):
-        return self._rebuild(*args, evaluate=False)
+    def func(self, *args, **kwargs):
```

Comment:
since it's a Reconstructable, and in particular a Pickable, I think that the following should suffice:

```
func = Pickable._rebuild
```